src/mutation_data.cpp

```diff
@@ -676,10 +678,13 @@ void mutation_branch::check_consistency()
         if( s_id && !s_id.value().is_valid() ) {
             debugmsg( "mutation %s refers to undefined scent type %s", mid.c_str(), s_id.value().c_str() );
         }
+        // Suppress these onload warnings for overlapping mods
         for( const trait_id &replacement : mdata.replacements ) {
             const mutation_branch &rdata = replacement.obj();
+            bool suppressed = rdata.src.back().second != trait_source && !basegame_trait;
             for( const mutation_category_id &cat : rdata.category ) {
-                if( std::find( mdata.category.begin(), mdata.category.end(), cat ) == mdata.category.end() ) {
+                if( std::find( mdata.category.begin(), mdata.category.end(), cat ) == mdata.category.end()  &&
```

Comment:
```suggestion
                if( std::find( mdata.category.begin(), mdata.category.end(), cat ) == mdata.category.end() &&
```
I am surprised it doesn't require a single space.

No need to include this change, I would just think the bot would have complained.

Reply:
Or did you bypass the bot style by going from draft to ready for review?

Reply:
I wait till the tests give the thumbsup and I'll clean it up.

Reply:
Astyle never complains about extra white spaces, since they're usually intentional choices to increase readability.